metricshub-doc/src/site/markdown/release-notes.md

```diff
@@ -5,6 +5,81 @@ description: Learn more about the new features, changes and improvements, and bu
 
 <!-- MACRO{toc|fromDepth=1|toDepth=1|id=toc} -->
 
+## MetricsHub Enterprise Edition v3.0.01
+
+### MetricsHub Enterprise Edition v3.0.01
+
+#### Changes and Improvements
+
+| ID                                                                          | Description                                                 |
+| --------------------------------------------------------------------------- | ----------------------------------------------------------- |
+| [**\#98**](https://github.com/MetricsHub/metricshub-enterprise/issues/98)   | Added a `user` CLI to manage REST API users                 |
```

Comment:
@NassimBtk: can you provide more information about value the issue #98 brings to the final user. The issue description is succinct and I could not find any information in the user doc.

Reply:
@NassimBtk, what about:

Added a CLI tool to securely create, list, and delete REST API users for JWT-based authentication.